src/test/java/com/getindata/connectors/http/internal/table/lookup/querycreators/ElasticSearchLiteQueryCreatorTest.java

```diff
@@ -18,6 +18,24 @@
 
 public class ElasticSearchLiteQueryCreatorTest {
 
+    @Test
```

Comment:
I think these new tests are not related to the issue HTTP-119? Could you please add them in a separate PR?

Reply:
Agreed - it would be good to have unit tests to test the various code paths around this change.

Reply:
The "new code" is already partially covered by existing tests, but I will try to add new ones to cover it fully.

The reason why I added those new tests was that we dropped with branch coverage below 90% and while inspecting test report I have found those two.

I will like to leave those tests in this PR, unless there will be a strong NO to it. The PR is small so IMO having those extra, unrelated tests is ok.

![image](https://github.com/user-attachments/assets/f6cff1f0-3ddf-40a9-a662-b44cc24f2e9b)

Reply:
That's fine for me.